text/src/test/scala/akka/stream/alpakka/text/scaladsl/CharsetCodingFlowsSpec.scala

```diff
@@ -77,7 +74,7 @@ class CharsetCodingFlowsSpec
           .via(TextFlow.encoding(StandardCharsets.US_ASCII))
           .intersperse(ByteString("\n"))
           .runWith(FileIO.toPath(targetFile))
-      result.futureValue.status should be(Success(Done))
+      result.futureValue.count should be > 50L
```

Comment:
What's the purpose of the `50L` constant? When I ran this test myself the count was `7430`.

Reply:
The input is system dependent and not stable.

Reply:
Wouldn't it be safer to rely on `status` then?  How did you arrive at `50`?

Reply:
That's where this began: `status` is deprecated in Akka 2.6. If the future completes status is always `Success(Done)`...
To add some other validation I randomly picked 50 to show that some data came through. It really doesn't matter.